code/obj/item/staffkiosk.dm

```diff
@@ -0,0 +1,287 @@
+	cardIcon = "id_eng"
+	tokenIcon = "assist-token-mine"
+	dispensedKit = /obj/item/storage/box/staffkit/miner
+
+/datum/recruitment_role/sci
+	name = "Research Assistant"
+	accessParent = "Scientist"
+	canAuthorize = list("Scientist","Research Director","Head of Personnel","Captain")
+	cardIcon = "id_res"
+	tokenIcon = "assist-token-sci"
+	dispensedKit = /obj/item/storage/box/staffkit/res
+
+/datum/recruitment_role/chef
+	name = "Culinary Assistant"
+	accessParent = "Chef"
+	canAuthorize = list("Chef","Head of Personnel","Captain")
+	cardIcon = "id_civ"
+	tokenIcon = "assist-token-chef"
+	dispensedKit = /obj/item/storage/box/staffkit/chef
+
+/datum/recruitment_role/bar
+	name = "Bar Assistant"
+	accessParent = "Bartender"
+	canAuthorize = list("Bartender","Head of Personnel","Captain")
+	cardIcon = "id_civ"
+	tokenIcon = "assist-token-bar"
+	dispensedKit = /obj/item/storage/box/staffkit/bar
+
+/datum/recruitment_role/botany
+	name = "Botanical Assistant"
+	accessParent = "Botanist"
+	canAuthorize = list("Botanist","Head of Personnel","Captain")
+	cardIcon = "id_civ"
+	tokenIcon = "assist-token-hyd"
+	dispensedKit = /obj/item/storage/box/staffkit/botany
+
+
+//equipment kits
+
+/obj/item/storage/box/staffkit/med
+	name = "medical assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffmed,
+		/obj/item/device/radio/headset/medical,
+		/obj/item/disk/data/cartridge/medical,
+		/obj/item/reagent_containers/emergency_injector/epinephrine,
+		/obj/item/reagent_containers/mender/brute,
+		/obj/item/reagent_containers/mender/burn,
+		/obj/item/bandage)
+
+/obj/item/storage/box/staffkit/robo
+	name = "robotics assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffmed,
+		/obj/item/device/radio/headset/medical,
+		/obj/item/disk/data/cartridge/medical,
+		/obj/item/screwdriver,
+		/obj/item/cable_coil,
+		/obj/item/body_bag)
+
+/obj/item/storage/box/staffkit/eng
+	name = "engineering assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffeng,
+		/obj/item/device/radio/headset/engineer,
+		/obj/item/disk/data/cartridge/engineer,
+		/obj/item/clothing/head/helmet/hardhat,
+		/obj/item/crowbar,
+		/obj/item/screwdriver,
+		/obj/item/device/multitool)
+
+/obj/item/storage/box/staffkit/mech
+	name = "mechanics assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffeng,
+		/obj/item/device/radio/headset/engineer,
+		/obj/item/disk/data/cartridge/mechanic,
+		/obj/item/electronics/soldering,
+		/obj/item/device/t_scanner,
+		/obj/item/screwdriver,
+		/obj/item/device/multitool)
+
+/obj/item/storage/box/staffkit/miner
+	name = "mining assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffeng,
```

Comment:
`/obj/item/paper/book/pocketguide/mining1`
`/obj/item/paper/book/pocketguide/mining2`

Reply:
Kit's full. Can fit them by losing the mining satchel and the crowbar, will probably do that (neither are outright essential for mining and both can be fairly easily manufactured).